yarn.lock

```diff
@@ -49,14 +49,14 @@ __metadata:
   version: 0.0.0-use.local
   resolution: "@apollo/client-react-streaming@workspace:packages/client-react-streaming"
   dependencies:
-    "@apollo/client": "npm:3.9.9"
+    "@apollo/client": "npm:3.10.1"
```

Comment:
We might want to get https://github.com/apollographql/apollo-client/pull/11821 out before we go live with the change in `useTransportedQueryRef` just to make sure its still compatible with the change in that PR.

Reply:
This wraps around useReadQuery and useQueryRefHandlers, and it can also create a queryRef by reviving a transportedQueryRef. We can reduce these effects once React deduplicates transported objects, but for now they're fixed.